Running-acceptance-tests.md

```diff
@@ -133,6 +133,13 @@ Successful on 1 target: localhost:2222
 Ran on 1 target in 1.72 sec
 ```
 
+If you want to install a specific version of puppet you can use the following command:
+```
+pdk bundle exec rake 'litmus:install_agent[puppet6]
+```
+Examples of other versions you can pass in are: puppet6-nightly, puppet7, puppet7-nightly.
+
+
 ## 5. Install the MoTD module on the CentOS image.
```

Comment:
Suggest adding something here like the following as it was not immediately apparent that this step was needed every time code changes.

```
If you are interactively modifying code and testing, note that this step must be run after changing the code before the acceptance tests are run.
```